R/importer.R

```diff
@@ -23,3 +23,15 @@ urltools::url_decode
 anonymize <- function(vec, algo = "sha256", seed = 0, chars = letters, n_chars = 5L, ...){
   anonymizer::anonymize(vec, .algo = algo, .seed = seed, .chars = chars, .n_chars = n_chars, ...)
 }
+
+#' @importFrom tidytext tidy
+#' @export
+tidy
```

Comment:
Actually, I'm not really sure about behaviours around here but I needed this way of definition for tidy.xgb.Booster